dj_cqrs/utils.py

```diff
@@ -0,0 +1,41 @@
+#  Copyright © 2021 Ingram Micro Inc. All rights reserved.
+
+import logging
+from datetime import datetime, timedelta, timezone
+
+from django.conf import settings
+
+
+logger = logging.getLogger('django-cqrs')
+
+
+def get_expires_datetime():
+    """Calculates when message should expire.
+
+    :return: datetime instance, None if infinite
+    :rtype: datetime
+    """
+    default_message_ttl = 86400  # 1 day
```

Comment:
Constants?

Reply:
Thought default settings in constants it's not the best solution.
But if you ok, i will ok too.